actions/update-charts-pr/action.yaml

```
@@ -0,0 +1,78 @@
+name: "Update site-values and create PR"
+description: "Checks out the site-values repo, updates the chart tag for the current service, and opens a PR"
+
+inputs:
+  tag:
+    description: "Chart tag (version) to set for this service"
+    required: true
+  paths:
+    description: "One or more YAML paths to update (newline or comma separated)"
+    required: true
+  chart:
+    description: "Name of the service repository to update in site-values"
+    required: false
+    default: ${{ github.event.repository.name }}
+  github_token:
+    description: "GitHub token with write access to the site-values repository"
+    required: true
+  repository:
+    description: "Target repository"
+    required: false
+    default: "site-values"
+  branch:
+    description: "Target base branch in repository"
+    required: false
+    default: "master"
+  pr_labels:
+    description: "Labels to apply on the created PR (newline or comma separated)"
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout ${{ inputs.repository }} repo
+      uses: actions/checkout@v5
+      with:
+        repository: MapColonies/${{ inputs.repository }}
+        token: ${{ inputs.github_token }}
+        path: ${{ inputs.repository }}
+        ref: ${{ inputs.branch }}
+
+    - name: Update tags in ${{ inputs.repository }}
+      uses: mikefarah/yq@v4.48.2
+      with:
+        cmd: |
+          set -euo pipefail
+
+          echo "Updating chartsVersions[\"${{ inputs.chart }}\"] to \"${{ inputs.tag }}\" in:"
+          # Support newline or comma separated list
+          echo '${{ inputs.paths }}' | tr ',' '\n' | while IFS= read -r relpath; do
+            # skip empty lines
+            [ -z "$relpath" ] && continue
+
+            # trim possible leading/trailing spaces
+            relpath="$(echo "$relpath" | xargs)"
+
+            echo "  - $relpath"
+            yq eval -i \
+              ".chartsVersions[\"${{ inputs.chart }}\"] = \"${{ inputs.tag }}\"" \
+              "${{ inputs.repository }}/$relpath"
+          done
+
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v6
+      with:
+        path: ${{ inputs.repository }}
+        token: ${{ inputs.github_token }}
+        commit-message: >-
+          feat: update ${{ inputs.chart }} to ${{ inputs.tag }}
+        title: >-
+          feat: update ${{ inputs.chart }} to ${{ inputs.tag }}
+        body: |
+          Trigger commit: https://github.com/${{ github.repository }}/commit/${{ github.sha }}
+
+          Updated files:
+          ${{ inputs.paths }}
+        labels: ${{ inputs.pr_labels }}
+        branch: ${{ inputs.chart }}-${{ github.sha }}
```

It is best to allow updating an existing branch and not always create a new one (in case the auto merge won't be triggered)
```suggestion
        branch: ${{ inputs.chart }}-${{ inputs.tag }}
```